packages/docusaurus-theme-classic/src/theme/Navbar/index.tsx

```diff
@@ -155,12 +156,22 @@ function NavbarMobileSidebar({
           imageClassName="navbar__logo"
           titleClassName="navbar__title"
         />
-        {!colorModeToggle.disabled && sidebarShown && (
+        {!colorModeToggle.disabled && (
           <Toggle
             checked={colorModeToggle.isDarkTheme}
```

Comment:
We need to add custom class for mobile toggle with `margin-right: 1rem` to avoid this issue:

![image](https://user-images.githubusercontent.com/4408379/131699964-56088eb1-7586-4f94-a756-bc58cd05b5f7.png)